components/script/dom/document.rs

```diff
@@ -247,6 +248,9 @@ pub struct Document {
     referrer: Option<String>,
     /// https://html.spec.whatwg.org/multipage/#target-element
     target_element: MutNullableHeap<JS<Element>>,
+    /// https://w3c.github.io/uievents/#event-type-dblclick
+    #[ignore_heap_size_of = ""]
```

Comment:
jdm reckons that `std::time` types should not be added to servo/heapsize.  Should I just set the `ignore_heap_size_of` to the URL of the issue anyway?